### PR TITLE
Prepare for release v0.8.1

### PR DESCRIPTION
**Pull request by felipecsl**
@spikebrehm @lelandrichardson

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+### Patches
```
Comment (spikebrehm) [inline CHANGELOG.md]: Can you add:

```
## 0.8.1 (September 8, 2016)
```

**Comment by spikebrehm** [comment]
LGTM!

**Comment by stereodenis** [comment]
@spikebrehm 
Got this
I try to remove node_modules and reinstall by npm i, but got the same


```
Failed to build DependencyGraph: @providesModule naming collision:
  Duplicate module name: String.prototype.es6
  Paths: /Users/stereodenis/work/stereopark/PshPshMobile/node_modules/react-native/packager/react-packager/src/Resolver/polyfills/String.prototype.es6.js collides with /Users/stereodenis/work/stereopark/PshPshMobile/node_modules/react-native-maps/example2/node_modules/react-native/packager/react-packager/src/Resolver/polyfills/String.prototype.es6.js

This error is caused by a @providesModule declaration with the same name across two different files.
Error: @providesModule naming collision:
  Duplicate module name: String.prototype.es6
  Paths: /Users/stereodenis/work/stereopark/PshPshMobile/node_modules/react-native/packager/react-packager/src/Resolver/polyfills/String.prototype.es6.js collides with /Users/stereodenis/work/stereopark/PshPshMobile/node_modules/react-native-maps/example2/node_modules/react-native/packager/react-packager/src/Resolver/polyfills/String.prototype.es6.js

This error is caused by a @providesModule declaration with the same name across two different files.
    at HasteMap._updateHasteMap (/Users/stereodenis/work/stereopark/PshPshMobile/node_modules/react-native/packager/react-packager/src/node-haste/DependencyGraph/HasteMap.js:155:13)
    at module.getName.then.name (/Users/stereodenis/work/stereopark/PshPshMobile/node_modules/react-native/packager/react-packager/src/node-haste/DependencyGraph/HasteMap.js:115:31)
```

**Comment by spikebrehm** [comment]
@stereodenis oh no! My bad.  I'll publish 0.8.2 to fix this.

https://github.com/lelandrichardson/react-native-maps/pull/558

**Comment by spikebrehm** [comment]
Just published [0.8.2](https://github.com/lelandrichardson/react-native-maps/releases/tag/v0.8.2).

**Pull request merged**